docs/mast/mast.rst

```diff
@@ -175,124 +174,6 @@ To get a table of metadata associated with observation or product lists use the
    obs_collection          Mission ... HST, HLA, SWIFT, GALEX, Kepler, K2...
 
 
-
-Mission Searches
```

Comment:
Because it was in between the Mast Observations section and the Downloading Data section, which are related because you download data from the observations queries, whereas the Missions interface is completely different and does not yet hook into the download data functionality.

Reply:
OK. Let's merge #2321 then first (it's almost there) and then do this reorder.